requirements.txt

```diff
@@ -5,3 +5,6 @@ django-tenants==2.0.0
 unittest-xml-reporting==2.2.0
 ddtrace==0.12.1
 raven==6.9.0
+djangorestframework
+markdown
+django-filter
```

Comment:
Please pin all to latest available on pypi.

Also, why are we requiring `markdown`?

Reply:
I _think_ that goes with DRF for the web-browsable half of the API.

Reply:
The last two are optional per DRF docs. I'm okay keeping them, just wanted to double-check on why they were there.

Still pin the versions.